File: mpwrapper/src/main/java/com/wgyscsf/mpwrapper/bean/Vol.java
package com.wgyscsf.mpwrapper.bean;

/**
 * ============================================================
 * 作 者 :    wgyscsf@163.com
 * 更新时间 ：2018/09/14 15:28
 * 描 述 ：量图，对于量图有两条ma线，作用和主图的ma基本一致。
 * ============================================================
 */
public class Vol {
    //vol 量，可选
    public double vol = KViewConstant.VALUE_DEF;
    //vol
    public double volMa5 = KViewConstant.VALUE_DEF;
    public double volMa10 = KViewConstant.VALUE_DEF;

    @Override
    public String toString() {
        return "Vol{" +
                "vol=" + vol +
                ", volMa5=" + volMa5 +
                ", volMa10=" + volMa10 +
                '}';
    }
}

File: mpwrapper/src/main/java/com/matt/mpwrapper/bean/Vol.kt
package com.matt.mpwrapper.bean

/**
 * ============================================================
 * 作 者 :    matt@163.com
 * 更新时间 ：2018/09/14 15:28
 * 描 述 ：量图，对于量图有两条ma线，作用和主图的ma基本一致。
 * ============================================================
 */
data class Vol(var vol: Float, val volMa5: Float, val volMa10: Float)